site/_posts/2023-07-11-get-more-value-out-of-fusionauth.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: blog-post
+title: Get More Value Out of FusionAuth Using the APIs for User Maintenance
+description: Customizing the order for your favorite food can make a good meal better. Customizing your end-users experience with user maintenance can make their experience with your application even better.
+author: Mark Robustelli
+image: blogs/get-more-value/get-more-value-header.png
+category: article
+tags: api user maintenance windows forms c#
+excerpt_separator: "<!--more-->"
+---
+
+So, you have just set up FusionAuth and are excited to take your new [customer identity and access management](/articles/ciam/) platform out for a spin. You spend some time setting up your Users, Applications, Tenants, etc. in the robust admin interface provided. Things are looking great. You have integrated your applications and are humming along. Now comes a real test, user maintenance.
+
+<!--more-->
+
+## Introduction
+*“A new user with extra pickles, please.”*
+
+Customizing the order for your favorite food can make a good meal better. Customizing your end-users' experience with user maintenance can make their experience with your application even better.
+
+For any [secure application](/articles/security/steps-secure-your-authentication-system), user maintenance is a must. As a developer, you can often overlook the time and effort it takes to maintain the users access and permissions for the shiny new application you created. In an ideal world, you do not want to be the ones to maintain the users. You want to give the task of user maintenance to the power users of the applications by making them application administrators. While those power users may be great at using the application, you don’t necessarily want to make them expert users of the FusionAuth admin interface. Well, we at FusionAuth would think it would be great if everyone was a master with it, but you may not share that same vision.
+
+Imagine you have an applicaiton that controls a machine that applies the toppings to a burger. Custom Buger, the popular new burger joint just bought your application. The tech-savvy manger, Sally Power-User, has been given admin rights. Sally, just hired Joe Limited-Access to work the night shift. She now needs to add Joe as a user of your application. She needs to ensure Joe can add pickles to a burger order but make sure he does not have access to the apply the famous Flaming Carolina Reaper Sauce until he goes through training. Your application will have to support this.
```

Comment:
```suggestion
Imagine you have an applicaiton that controls a machine that applies the toppings to a burger. Custom Burger, the popular new burger joint just bought your application. The tech-savvy manger, Sally Power-User, has been given admin rights. Sally, just hired Joe Limited-Access to work the night shift. She now needs to add Joe as a user of your application. She needs to ensure Joe can add pickles to a burger order but make sure he does not have access to the apply the famous Flaming Carolina Reaper Sauce until he goes through training. Your application will have to support this.
```

Please copy and paste this post into google docs and run spell/grammar check on it.

Reply:
Done.  Rookie mistake.